pyrealm_build_data/synth_data.py

```diff
@@ -0,0 +1,81 @@
+"""This script uses a parametrized model to compress the input dataset.
+
+It fits a time series model to the input data and stores the model parameters.
+The dataset can then be reconstructed from the model parameters using the `reconstruct`
+function, provided with a custom time index.
+"""
```

Comment:
What is this custom time index and the special time features - not clear.

Reply:
It's just any array of the `datetime64` data type or a pandas `DatetimeIndex`, possibly generated by something like `pd.date_range("2000-1-1", "2009-12-31", freq="1h")`.